Fluent/Post_Process.ipynb

```diff
@@ -6,127 +6,63 @@
    "source": [
     "# CFD Results Post-Process & Validation\n",
     "\n",
-    "< copy and paste the two below paragraphs on top of your Post_Process file. Remove this guideline after that. >\n",
-    "\n",
     "**The final step for the development of a CFD simulation for a problem of interest is to process and examine the validity of the obtained numerical results (a.k.a post-processing and validation step). These steps requires an in-depth engineering knowledge about the fundamentals of the physical problem of interest. Thus, the value of investing time in understanding the fundamental physics of the flow, once again is highlighted here. This process is done starting from general inspection on CFD simulation results and ends with more detail validation process.**\n",
     "\n",
+    "#### 1- Start with visualization of the residuals for most important variables within the flow field (e.g. continuity, streamwise velocity and etc.). Discuss how user should examine the residuals trends to comment on accuracy and convergence of the flow field's simulation. \n",
     "\n",
-    "# General Recommended Structure for Writing Post_Process File\n",
+    "This section discusses the results of the simulation for ** 2D Flow Over Rotating Cylinder**. The Reynoldsnumber is 100. The first check point in any CFD Simulation is to check the convergence of residuals of the variables. The variables can be 'Continuity', 'Momentum' etc. Residuals are differences in the value of a quantity between two iterations. A low value of residual indicates that the solution is converging. It means that futher iterations will have very small effect on the solution and the solution iteration can be stopped. Another way of knowing whether tyhe solution has converged is to look at the values of Lift coefficient (Cl) and Drag coefficient (Cd) at each timestep. The Cl must attain a constant value or oscillate about a mean value. Same goes for Cd. The plot for Cl is shown below.\n",
     "\n",
```

Comment:
Split this paragraph into two sections:

1-Talk about the residual values of the chosen variables and show a plot of that. Describe exactly what the plot shows and how you interpret it.

2-Show plot of convergence history for C_L. If you are not visualizing history of C_D, briefly explained how it looks like.

Make sure to put the definitions of figure variables and discussion of the trends before figure, not after it. This would motivated and guide your readers.